<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_seller</class>
 <widget class="QDialog" name="New_seller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прізвище, ім'я:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IBAN:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Банк:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ЄДРПОУ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ІПН:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Адреса:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="line_name_3">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прізвище</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_iban">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IBAN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_bank">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Банк</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_edrpoy">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ЄДРПОУ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_ipn_3">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ІПН</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_address">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Адреса</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Додати</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reject_3">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
